app/views/content_items/world_location_news_article.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, @content_item.page_title %>
+<%= content_for :meta_description, @content_item.description %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: t("content_item.format.#{@content_item.document_type}", count: 1),
+        title: @content_item.title,
+        average_title_length: "long" %>
+  </div>
+
+  <div class="column-third">
+    <%= render 'shared/available_languages',
+      translations: @content_item.available_translations
+    %>
+  </div>
+</div>
```

Comment:
This block should use the `title_and_translations` partial along with the TitleAndContext module in the presenter.